modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/DistributionZoneManager.java

```diff
@@ -192,7 +217,7 @@ public CompletableFuture<Void> alterZone(String name, DistributionZoneConfigurat
         Objects.requireNonNull(distributionZoneCfg, "Distribution zone configuration is null.");
 
         if (!busyLock.enterBusy()) {
-            throw new IgniteException(new NodeStoppingException());
+            throw new IgniteException(NODE_STOPPING_ERR, new NodeStoppingException());
```

Comment:
What's the point of given code?

Reply:
fixed in https://github.com/gridgain/apache-ignite-3/pull/85